.gitignore

```diff
@@ -0,0 +1,6 @@
+config/database.yml
+/log
+/tmp
+
+# Only include if you have production secrets in this file, which is no longer a Rails default
+config/secrets.yml
```

Comment:
This file is also added in your PR, but you can keep it in your repo, since it doesn't have production secrets.

Reply:
done